pkgs/by-name/zd/zdns/package.nix

```diff
@@ -6,16 +6,16 @@
 
 buildGoModule rec {
   pname = "zdns";
-  version = "2023-04-09-unstable";
+  version = "2.0.0";
 
   src = fetchFromGitHub {
     owner = "zmap";
     repo = pname;
-    rev = "ac6c7f30a7f5e11f87779f5275adeed117227cd6";
-    hash = "sha256-que2uzIH8GybU6Ekumg/MjgBHSmFCF+T7PWye+25kaY=";
+    rev = "c889e07be3b4991f5ef67c41ad1c0f76e5a2b4df";
```

Comment:
```suggestion
    tag = "v${version}";
```

Reply:
Fixed while adjusting the commit message. Thanks for the pointer!